.gitmodules

```diff
@@ -4,3 +4,9 @@
 [submodule ".cbmc-batch/aws-c-common"]
 	path = verification/cbmc/aws-c-common
 	url = https://github.com/awslabs/aws-c-common.git
+[submodule "verification/cbmc/templates"]
+	path = verification/cbmc/templates
+	url = https://github.com/awslabs/aws-templates-for-cbmc-proofs.git
+[submodule "verification/litani"]
+	path = verification/litani
+	url = https://github.com/awslabs/aws-build-accumulator
```

Comment:
Why is this URL missing ".git" at the end? Mentioning it here just in case.

Reply:
It works either way